test-requirements.txt

```diff
@@ -1,2 +1,2 @@
-git+git://github.com/gtmanfred/testinfra.git@master#egg=testinfra
+git+git://github.com/philpep/testinfra@master#egg=testinfra
 paramiko
```

Comment:
on my workstation is also pytest-catchlog missing (ImportError: Error importing plugin "catchlog": No module named catchlog)